mod_app/static/css/bibliography.css

```diff
@@ -1,19 +1,26 @@
 .bibliography__area {
-  margin: 2rem 1rem;
-}
-.bibliography__item {
-  margin-bottom: 1rem;
-  display: flex;
-  columns: 2;
+  margin: 1vw 6vw;
```

Comment:
It might just be personal bias but I felt like it worked a bit better for responsiveness - maybe since it's based off the width of the screen instead of an element?